### PR TITLE
fix: fixed some formats

### PR DESCRIPTION
**Pull request by twchn**

### CODE REVIEW

File: src/getting-started/choose-web-library.md [hunk 1]
```diff
@@ -115,7 +115,6 @@ There are a few different angles to consider when choosing between using `web-sy
       </td>
     </tr>
   </tbody>
-</table>For more information about picking a build tool, checkout the following page:
-
-
+</table>
 
+For more information about picking a build tool, checkout the following page:
```
Comment (twchn) [inline src/getting-started/choose-web-library.md:120]: I noticed that there is a link below in the document website, but it is not shown here. Why? Is it controlled by this non-newline code?

File: src/getting-started/project-setup/using-cargo-web.md [hunk 2]
```diff
@@ -27,6 +27,6 @@ cargo web start
 * `asmjs-unknown-emscripten`
 
 {% hint style="info" %}
-For `*-emscripten` targets, you'll need to install the Emscripten SDK
+For `*-emscripten` targets, `cargo-web` will automatically install the Emscripten SDK and generate targets for you.
```
Comment (twchn) [inline src/getting-started/project-setup/using-cargo-web.md:30]: This sentence is different from the `zh_cn` branch, as `zh_cn` branch is more detailed, I just translate it into English.

Comment (jstarry) [reply src/getting-started/project-setup/using-cargo-web.md:30]: Ah thanks but actually this sentence is only correct for Linux, so I removed it. Can you please update the `zh_cn` to match the `master` branch?

Comment (twchn) [reply src/getting-started/project-setup/using-cargo-web.md:30]: My pleasure.

File: src/getting-started/choose-web-library.md (hunk 1 above)
Comment (jstarry) [reply src/getting-started/choose-web-library.md:120]: Sometimes Gitbook does strange things that I don't understand 😅 Anyways, the link on Gitbook was wrong so I just updated it to be an inline link so this shouldn't be an issue anymore. Can you rebase?

**Pull request closed without merging**